components/console-backend-service/internal/domain/servicecatalog/serviceplan_resolver.go

```diff
@@ -0,0 +1,62 @@
+package servicecatalog
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/domain/shared"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlerror"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+)
+
+type servicePlanResolver struct {
+	sc     *servicePlanService
+	rafter shared.RafterRetriever
+}
+
+func newServicePlanResolver(sc *servicePlanService, r shared.RafterRetriever) *servicePlanResolver {
+	return &servicePlanResolver{sc, r}
+}
+
+func (r *servicePlanResolver) ServicePlanClusterAssetGroupField(ctx context.Context, obj *gqlschema.ServicePlan) (*gqlschema.ClusterAssetGroup, error) {
+	if obj == nil {
+		glog.Error(fmt.Errorf("while getting clusterAssetGroup field obj is empty"))
+		return nil, gqlerror.NewInternal()
+	}
+
+	assetGroup, err := r.rafter.ClusterAssetGroup().Find(obj.Name)
+	if err != nil {
+		glog.Errorf("Couldn't find clusterAssetGroup with name %s", obj.Name)
```

Comment:
Please use `pretty` form of types from `rafter` domain like here https://github.com/kyma-project/kyma/blob/master/components/console-backend-service/internal/domain/rafter/asset_service.go#L113 -> `pretty.ClusterAssetGroupType`. Please adjust it also in another places.